src/spec/determineConfig.spec.js

```diff
@@ -181,14 +209,18 @@ describe("determineConfig", function() {
       describe(
         `when args is ${JSON.stringify(example.args)} ` +
         `and config is ${JSON.stringify(example.config)} ` +
-        `and CHAI_JEST_SNAPSHOT_UPDATE_ALL is ${example.envFlag ? "set" : "not set"}`
+        `and CHAI_JEST_SNAPSHOT_UPDATE_ALL is ${example.envUpdateFlag ? "set" : "not set"} `+
+        `and CHAI_JEST_CI is ${example.envCiFlag ? "set" : "not set"}`
       , function() {
 
         const run = () => {
           delete process.env.CHAI_JEST_SNAPSHOT_UPDATE_ALL;
```

Comment:
You'll need to also delete the ci environment variable here, otherwise test cases will leak into each other